addons/web/static/src/views/relational_model.js

```diff
@@ -2367,10 +2367,18 @@ export class DynamicGroupList extends DynamicList {
      * @returns {Promise<Group>}
      */
     async _createGroup(groupName, groupData = {}, isFolded = false) {
-        groupData = { ...groupData, name: groupName };
-        const [id] = await this.model.orm.create(this.groupByField.relation, [groupData], {
-            context: this.context,
-        });
+        const [id] = await this.model.orm.call(
```

Comment:
problematic if (somehow) we can create a new column via a dialog. I'm not sure it exists though.